gems/pending/appliance_console/locales/container/en.yml

```diff
@@ -9,6 +9,7 @@ en:
     - db_config
     - db_replication
     - db_maintenance
+    - log_config
```

Comment:
I'm not sure how this would work for containers... @fbladilo do you have something else planned for this?